mffpy/reader.py

```diff
@@ -307,7 +326,7 @@ def get_mff_content(self):
         # Root tags corresponding to available XMLType sub-classes
         xml_root_tags = ['fileInfo', 'dataInfo', 'patient', 'sensorLayout',
                          'coordinates', 'epochs', 'eventTrack', 'categories',
-                         'dipoleSet']
+                         'dipoleSet', 'historyEntries']
```

Comment:
[required] Let's replace this with `list(xml_files.XMLType._tag_registry.keys())`.  Maybe it would be best if you write a function that returns these instead of accessing `_`-variables from the ABC.

Reply:
Good idea. This will make it so we don't have to update that list every time we add a new XML type.